hpcpy/client/base.py

```diff
@@ -248,9 +319,95 @@ def _render_directives(self, directives):
         str
             Rendered directives, or blank string.
         """
-
         # Render blank directives if empty
         if not directives:
             return ""
 
         return " " + " ".join(directives)
+
+    def hold(self, job_id):
+        """Hold a job.
+
+        Parameters
+        ----------
+        job_id : str
+            Job ID.
+
+        Returns
+        -------
+        str
+            Command response text.
+        """
+        cmd = self.cmd_templates["hold"].format(job_id=job_id)
+        result = self._shell(cmd)
+        return result
+
+    def release(self, job_id):
+        """Release a job.
+
+        Parameters
+        ----------
+        job_id : str
+            Job ID.
+
+        Returns
+        -------
+        str
+            Command response text.
+        """
+        cmd = self.cmd_templates["release"].format(job_id=job_id)
+        result = self._shell(cmd)
+        return result
+
+    def _assemble_delay_directive(self, delay) -> str:
+        """Assemble the delay directive for the scheduler.
+
+        Parameters
+        ----------
+        delay : datetime or timedelta
+            Either a specific datetime or delta from now.
+
```

Comment:
Suggestion as per above comment on `delay_directive_fmt`
```suggestion
    def _assemble_delay_directive(self, delay, delay_directive_fmt) -> str:
        """Assemble the delay directive for the scheduler.

        Parameters
        ----------
        delay : datetime or timedelta
            Either a specific datetime or delta from now.
        delay_directive_fmt : str
            Delay directive format.

```